ngraph/frontend/paddlepaddle/src/op/slice.cpp

```diff
@@ -35,24 +35,42 @@ NamedOutputs slice(const NodeContext& node) {
         end_idx_node = opset6::Constant::create(element::i32, {ends.size()}, ends);
     }
 
-    // the shape of input, such as [1, 1, 3, 3]
+    // The following process is:
```

Comment:
Do we have tests for the change?

Reply:
Yes, This change is covered by the original tests. This is a refinement. The latest error happens during the IR serialization, if we don't add zeros to the slice masks, then the IR file generated is illegal but the inference result is correct.